client/basic-authorship/src/basic_authorship.rs

```diff
@@ -386,6 +390,13 @@ where
 						MAX_SKIPPED_TRANSACTIONS - skipped,
 					);
 					continue
+				} else if (self.now)() < soft_deadline {
```

Comment:
```suggestion
				} else if now < soft_deadline {
```

Reply:
I wonder, isn't actually original  `(self.now)()` intentional to get more recent reading?

Reply:
It's a good suggestion, there is nothing intensive happening between these calls, and it's worth saving this one extra syscall. That was my intention when I introduced `now` variable initially as well, but somehow lost it in all the refactorings.